bedrock/settings/base.py

```diff
@@ -233,10 +235,10 @@ def JINJA_CONFIG():
     }
 
 
-MEDIA_URL = '/user-media/'
-MEDIA_ROOT = path('media')
-STATIC_URL = '/media/'
-STATIC_ROOT = path('static')
+MEDIA_URL = config('MEDIA_URL', default='/user-media/')
```

Comment:
I'd say the media and static `ROOT` configs are among those unlikely to change, but this can't hurt I think.